src/cmdline.rs

```diff
@@ -136,6 +137,15 @@ pub fn parse_args() -> Result<Config> {
                         .help("Additional kernel args for the first boot")
                         .takes_value(true),
                 )
+                .arg(
+                    Arg::with_name("copy-network-config")
+                        .short("n")
+                        .long("copy-network-config")
+                        .value_name("path")
+                        .default_value("/etc/NetworkManager/system-connections/")
+                        .takes_value(true)
+                        .help("Optionally copy the network config from <path> in the installer environment"),
```

Comment:
This breaks the 80-column limit on readable help text.  `Copy network config from <path>` almost fixes it, but the option description (`--copy-network-config <path>`) is so wide that it shifts _other_ options' help text to the right.

Which intersects with another issue: I don't think the single option is going to work.  `coreos-installer install -n /dev/loop0` and `coreos-installer install --copy-network-config /dev/loop0` both misinterpret `/dev/loop0` as the path to the network configs.  Without a more discerning parser, I think we should split this into two options: a boolean, and an option to override the default path.

Dropping `<path>` from the option description would fix the help text length without needing to rename the option, though it then moves the problem to choosing the name of the second option.

I know the character-counting is painful but we don't really get usable help text otherwise.  We could enable the `wrap_help` clap feature but it's... only slightly less bad:

```
Install Fedora CoreOS or RHEL CoreOS

USAGE:
    coreos-installer install [OPTIONS] <device>

OPTIONS:
    -s, --stream <name>                 Fedora CoreOS stream [default: stable]
    -u, --image-url <URL>               Manually specify the image URL
    -f, --image-file <path>             Manually specify a local image file
    -i, --ignition-file <path>          Embed an Ignition config from a file
    -p, --platform <name>               Override the Ignition platform ID
        --firstboot-args <args>
            Additional kernel args for the first boot

    -n, --copy-network-config <path>
            Optionally copy the network config from <path> in the installer
            environment [default: /etc/NetworkManager/system-connections/]
        --insecure                      Skip signature verification
        --stream-base-url <URL>
            Base URL for Fedora CoreOS stream metadata

        --architecture <name>
            Target CPU architecture [default: x86_64]

        --preserve-on-error             Don't clear partition table on error
    -h, --help                          Prints help information

ARGS:
    <device>    Destination device
```

Reply:
> This breaks the 80-column limit on readable help text. `Copy network config from <path>` almost fixes it, but the option description (`--copy-network-config <path>`) is so wide that it shifts _other_ options' help text to the right.
> 
> Which intersects with another issue: I don't think the single option is going to work. `coreos-installer install -n /dev/loop0` and `coreos-installer install --copy-network-config /dev/loop0` both misinterpret `/dev/loop0` as the path to the network configs. Without a more discerning parser, I think we should split this into two options: a boolean, and an option to override the default path.

I'll work on adding the second option.

> Dropping `<path>` from the option description would fix the help text length without needing to rename the option, though it then moves the problem to choosing the name of the second option.

I'll try to get the text to under 80 characters.

> 
> I know the character-counting is painful but we don't really get usable help text otherwise. We could enable the `wrap_help` clap feature but it's... only slightly less bad:

On a console readability within 80 characters in important.

Reply:
The closest I could get was changing the long form of `-n` to `--copy-network`, adding a `--network-dir` option and using `.next_line_help(true)`. Here's what we end up with, which is less than 80 characters and I think a good compromise:

```
Install Fedora CoreOS or RHEL CoreOS

USAGE:
    coreos-installer install [OPTIONS] <device>

OPTIONS:
    -s, --stream <name>            Fedora CoreOS stream [default: stable]
    -u, --image-url <URL>          Manually specify the image URL
    -f, --image-file <path>        Manually specify a local image file
    -i, --ignition-file <path>     Embed an Ignition config from a file
    -p, --platform <name>          Override the Ignition platform ID
        --firstboot-args <args>    Additional kernel args for the first boot
    -n, --copy-network             Copy network config from install environment
        --network-dir <path>
            For use with -n. [default: /etc/NetworkManager/system-connections/]

        --insecure                 Skip signature verification
        --stream-base-url <URL>    Base URL for Fedora CoreOS stream metadata
        --architecture <name>      Target CPU architecture [default: x86_64]
        --preserve-on-error        Don't clear partition table on error
    -h, --help                     Prints help information

ARGS:
    <device>    Destination device
```

Reply:
We could avoid the second line with e.g. `Override default network config path` and `hide_default_value(true)`.

Reply:
I do think showing the user the default is important.